internal/config/config.go

```diff
@@ -46,8 +46,8 @@ type Config struct {
 
 	TunnelIP     net.IP       `desc:"IP to use for tunnels" split_words:"true"`
 	VxlanPort    uint16       `default:"0" desc:"VXLAN port to use" split_words:"true"`
-	VppAPISocket string       `default:"" desc:"filename of socket to connect to existing VPP instance.  If empty a VPP instance is run in forwarder" split_words:"true"`
-	VppInit      vppinit.Func `default:"AF_PACKET" desc:"type of VPP initialization.  Must be AF_PACKET or NONE" split_words:"true"`
+	VppAPISocket string       `default:"/var/run/vpp-ext/vpp-api.sock" desc:"filename of socket to connect to existing VPP instance.  If empty a VPP instance is run in forwarder" split_words:"true"`
```

Comment:
Question, why the vpp-ext as the default instead of vpp in the path?

Reply:
@edwarnicke 
Good point, thank you! Done